yarn-project/noir-contracts/src/contracts/lending_contract/src/storage.nr

```diff
@@ -53,35 +54,35 @@ fn serialiseTot(tot: Tot) -> [Field; TOT_SERIALISED_LEN] {
     [tot.interest_accumulator as Field, tot.last_updated_ts as Field]
 }
 
-global TotSerialisationInterface = TypeSerialisationInterface {
+global TotSerialisationMethods = TypeSerialisationInterface {
     deserialise: deserialiseTot,
     serialise: serialiseTot,
 };
 
 
-// Struct to be used to represent positions when we have more reads.
-global POS_SERIALISED_LEN: Field = 2;
+// // Struct to be used to represent positions when we have more reads.
```

Comment:
I didn't want to remove this `Pos` struct altogether yet, in case someone had grand designs. It can be removed in a future PR, if we don't see the need for it anymore.